app/Http/Controllers/Table/ProcessorsController.php

```diff
@@ -34,7 +34,7 @@ protected function baseQuery(Request $request): Builder
     {
         return Processor::query()
             ->hasAccess($request->user())
-            ->with(['device', 'device.location'])
+            ->leftJoin('devices', 'devices.device_id', '=', 'processors.device_id')
             ->withAggregate('device', 'hostname');
```

Comment:
Is `->withAggregate('device', 'hostname');` used anymore?

Reply:
Good spot, probably not,